examples/ansys-aedt-workflow/main.py

```diff
@@ -19,7 +19,7 @@
 scripts = workdir / "scripts"
 
 
-wb = launch_workbench(client_workdir=str(workdir.absolute()))
+wb = launch_workbench(version="241", client_workdir=str(workdir.absolute()))
```

Comment:
@jorgepiloto Please avoid a specific version here, rather we should let it use the default version, as the latter gets updated with new releases. 

The same goes for the other scripts in this pull request.

Reply:
The unified install in the VM must be updated then. Other option is to implement a solution in PyWorkbench that reads the version from an environment variable.

Reply:
@jorgepiloto Is it possible for you to install latest Ansys v25R1 unified build on VM? As new released packages of PyAnsys libraries are wrt Ansys v25R1. @franknli please correct us if it's otherwise.

Reply:
@jorgepiloto @vnamdeo  For now, we should install v242 on the VM - which is the default version. Once 251 is officially release, we will update the default version of pywb to 251.

Reply:
> @jorgepiloto @vnamdeo For now, we should install v242 on the VM - which is the default version. Once 251 is officially release, we will update the default version of pywb to 251.

@franknli, if this is the case, in that case, PyAnsys version need to align with following package versioning https://docs.pyansys.com/version/dev/package_versions/version_2024.2.html

Reply:
I am installing the latest versions in the VM.